services/speech_to_text/v1.js

```diff
@@ -99,10 +99,18 @@ module.exports = function (RED) {
 
     wstream.on('finish', function () {
       fs.readFile(file, function (err, buf) {
+        var fmt = null;
+        var error = null;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Combine this with the previous 'var' statement.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2697377886/issues/source?bid=3330694&fileBranchId=3341077#l103)